scripts/newinstall.sh

```diff
@@ -25,7 +25,7 @@ LSST_EUPS_PKGROOT_BASE_URL=${LSST_EUPS_PKGROOT_BASE_URL:-https://eups.lsst.codes
 LSST_EUPS_USE_TARBALLS=${LSST_EUPS_USE_TARBALLS:-false}
 LSST_EUPS_USE_EUPSPKG=${LSST_EUPS_USE_EUPSPKG:-true}
 
-LSST_MINICONDA_VERSION=${LSST_MINICONDA_VERSION:-4.7.12}
+LSST_MINICONDA_VERSION=${LSST_MINICONDA_VERSION:-4.8.3}
```

Comment:
This works? I thought they switched to using `py37-4.8.3` in the 4.8.x series?

Reply:
nope, it doesn't work :)